.github/workflows/ci.yml

```
@@ -83,24 +83,17 @@ jobs:
           echo "::set-output name=go-build::$(go env GOCACHE)"
           echo "::set-output name=go-mod::$(go env GOMODCACHE)"
 
-      # Cache go build cache, used to speedup go test
-      - name: Go Build Cache
-        uses: actions/cache@v2
+      - name: Go Cache
```

Not sure about this. The casing of names is inconsistent in this file.

```suggestion
      - name: Go cache
```